doc/source/management/data_at_rest_encryption.rst

```diff
@@ -11,244 +11,422 @@ This feature is considered **BETA** quality.
 
 .. _innodb_general_tablespace_encryption:
 
-InnoDB general tablespace encryption
-====================================
+Introduction
+============
 
-In |Percona Server| :rn:`5.7.20-18` existing tablespace encryption support
-has been extended to handle general tablespaces. A general tablespace is either
-fully encrypted, covering all the tables inside, either not encrypted at all.
-It is not possible to have only some of the tables in the general tablespace
-encrypted.
+"Data-at-rest" term refers to all the data stored on disk by some server within
+system tablespace, general tablespace, redo-logs, undo-logs, etc. As an
+opposite, "data-in-transit" means data transmitted to other node or client.
+Data-in-transit can be encrypted using SSL connection (details are available in
+the `encrypt traffic documentation <https://www.percona.com/doc/percona-xtradb-cluster/LATEST/security/encrypt-traffic.html>`_) and
+therefore supposed to be safe. Below sections are about securing the
+data-at-rest only. 
 
-This feature extends the  `CREATE TABLESPACE
-<https://dev.mysql.com/doc/refman/5.7/en/create-tablespace.html>`_
-statement to accept the ``ENCRYPTION='Y/N'`` option.
+Currently data-at-rest encryption is supported in |PXC| for general tablespace,
+file-per-tablespace, and temporary tables.
 
-Prerequisites
--------------
-
-This feature requires a keyring plugin, for example `keyring_file
-<https://dev.mysql.com/doc/refman/5.7/en/keyring-file-plugin.html>`_ or
-:ref:`keyring_vault_plugin` to be loaded before it can be used.
+InnoDB tablespace encryption
+============================
 
-.. warning::
+|MySQL| supports tablespace encryption, but only for file-per-table tablespace.
+User should create a table that has its own dedicated tablespace first, and
```

Comment:
User should create a table that has its own dedicated tablespace first, and
 +then can make this tablespace encrypted by specifying the appropriate option.

This sounds like 2 step process when it is just single step. Data-encryption is possible only for file-per-tablespace tables.